### PR TITLE
Rename bisheng to bishengjdk in domain

### PR DESCRIPTION
**Pull request by dongbohe**
Clarify the meaning of domain names

### CODE REVIEW

File: applications/openeuler-website-v2/ingress.yaml [hunk 1]
```diff
@@ -154,12 +154,12 @@ metadata:
   annotations:
     kubernetes.io/ingress.class: nginx
     cert-manager.io/cluster-issuer: letsencrypt-issuer
-    nginx.ingress.kubernetes.io/temporal-redirect: https://openeuler.org/zh/other/projects/bisheng/
-  name: bisheng-ingress
+    nginx.ingress.kubernetes.io/temporal-redirect: https://openeuler.org/zh/other/projects/bishengjdk/
+  name: bishengjdk-ingress
```
Comment (TommyLike) [inline applications/openeuler-website-v2/ingress.yaml:158]: No need to update this.

File: applications/openeuler-website-v2/ingress.yaml [hunk 2]
```diff
@@ -154,12 +154,12 @@ metadata:
   annotations:
     kubernetes.io/ingress.class: nginx
     cert-manager.io/cluster-issuer: letsencrypt-issuer
-    nginx.ingress.kubernetes.io/temporal-redirect: https://openeuler.org/zh/other/projects/bisheng/
-  name: bisheng-ingress
+    nginx.ingress.kubernetes.io/temporal-redirect: https://openeuler.org/zh/other/projects/bishengjdk/
```
Comment (TommyLike) [inline applications/openeuler-website-v2/ingress.yaml:157]: https://openeuler.org/zh/other/projects/bishengjdk/ is not found, is it correct?

Comment (dongbohe) [reply applications/openeuler-website-v2/ingress.yaml:157]: What can I do to use https://openeuler.org/zh/other/projects/bishengjdk/ ？

File: applications/openeuler-website-v2/ingress.yaml (hunk 1 above)
Comment (dongbohe) [reply applications/openeuler-website-v2/ingress.yaml:158]: Since this site has nothing to do with bisheng, so I want to use bishengjdk instead of bisheng

Comment (TommyLike) [reply applications/openeuler-website-v2/ingress.yaml:158]: ok

File: applications/openeuler-website-v2/ingress.yaml (hunk 2 above)
Comment (TommyLike) [reply applications/openeuler-website-v2/ingress.yaml:157]: let's wait until the url updated and corrected again.

**Review by TommyLike — APPROVED** [review]

**Pull request merged**